src/parsers/manifest/dash/parse_representations.ts

```diff
@@ -97,6 +101,37 @@ export interface IAdaptationInfos {
   unsafelyBaseOnPreviousAdaptation : Adaptation | null;
 }
 
+/**
+ * Extract HDR information from manifest and codecs.
+ * @param {Object}
+ * @returns {Object | undefined}
+ */
+function getHDRInformation(
+  { adaptationProfiles,
+    manifestProfiles,
+    codecs,
+  }: { adaptationProfiles?: string;
+       manifestProfiles?: string;
+       codecs?: string; }
+): undefined | IHDRInformation {
+  const profiles = (adaptationProfiles ?? "") + (manifestProfiles ?? "");
+  if (codecs === undefined) {
+    return undefined;
+  }
+  if (
+    profiles.indexOf(
+      "http://dashif.org/guidelines/dash-if-uhd#hevc-hdr-pq10") !== -1
+  ) {
+    if (codecs === "hvc1.2.4.L153.B0" ||
+        codecs === "hev1.2.4.L153.B0") {
+      return { colorDepth: 10,
+               eotf: "pq",
+               colorSpace: "rec2020" };
+    }
+  }
+  return getWEBMHDRProfile(codecs);
```

Comment:
So we call a function called "getWEBMHDRProfile" even if this is not a WEBM container?